package.json

```diff
@@ -2,38 +2,33 @@
   "name": "@ubiquity/dollar",
   "license": "MIT",
   "scripts": {
-    "ape": "run-s init:env make start",
-    "make": "run-s _contracts:install _contracts:build",
-    "start": "run-p _start:*",
-    "init:env": "cp .env.example .env",
-    "killscript": "kill `ps | grep dev-chain-init.sh | grep -v grep | awk '{ print $1 }'`",
-    "_start:chain": "./build-scripts/dev-chain-init.sh",
-    "_start:ui": "next dev",
+    "ape": "run-s install-env install-contracts dev",
```

Comment:
I will eventually want to make formal/standard script names; perhaps `yarn start` could be more appropriate